docs/index.rst

```diff
@@ -229,6 +229,8 @@ Ready to start reviewing pull requests? We have :doc:`a guide </reviewing_PRs>`
 for that too! Finally, the :doc:`API reference </api/index>` covers individual
 functions, classes, methods and modules.
 
+**Robottelo** is compatible with Python 2.7.
```

Comment:
@rplevka @JacobCallahan @lpramuk I remember some of you talking about using Python 3 on testing? should we include Python 3.5 as compatible?